otcextensions/osclient/dns/v2/zone.py

```diff
@@ -86,15 +86,28 @@ def get_parser(self, prog_name):
             help=_('UUID or name of the zone.')
         )
 
+        parser.add_argument(
+            '--type',
+            metavar='<type>',
+            choices=['private'],
```

Comment:
here can be only one choice?

Reply:
Yes, this is not an issue. It is related due to the behavior of find and list method in OpenStack and the Huawei adaption.

Reply:
Hi Anton, I have changed the choices and description accordingly. Please check again.